### ISSUE TITLE
RM training loss becomes NAN when finish the first training step.

### ISSUE THREAD

**Issue by lixsh6** [OpenLLMAI/OpenRLHF#288]
I used a large model (> 170B) as my reward model. In the very beginning, loss is normal. But when training one step, the loss becomes NAN. This situation didn't happen when I used a smaller base model (e.g., 30B) to train RM. Do you have any suggestions about this?

**Comment by hijkzzz** [comment]
We don't have such a big model, maybe it is related to DeepSpeed?

**Comment by catqaq** [comment]
"But when training one step, the loss becomes NAN. "
more info about your data and configuration?
